benchmarks/array_transfer.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3                                                         
+
+import time, argparse
+import numpy as np
+import arkouda as ak
+
+TYPES = ('int64', 'float64')
+
+def time_ak_array_transfer(N, trials, dtype, random, seed):
+    print(">>> arkouda {} array creation".format(dtype))
+    cfg = ak.get_config()
+    print("numLocales = {}, N = {:,}".format(cfg["numLocales"], N))
+
+    a = ak.randint(0, 2**32, N, dtype=dtype, seed=seed)
+    nb = a.size * a.itemsize
+    ak.client.maxTransferBytes = nb
+
+    to_ndarray_times = [] 
+    to_pdarray_times = []
+    for i in range(trials):
+        start = time.time()
+        npa = a.to_ndarray();
+        end = time.time()
+        to_ndarray_times.append(end - start)
+        start = time.time();
+        aka = ak.array(npa)
+        end = time.time()
+        to_pdarray_times.append(end - start)
+    avgnd = sum(to_ndarray_times) / trials
+    avgpd = sum(to_pdarray_times) / trials
+
+    print("to_ndarray Average time = {:.4f} sec".format(avgnd))
+    print("ak.array Average time = {:.4f} sec".format(avgpd))
+
+    print("to_ndarray Average rate = {:.4f} GiB/sec".format(nb/2**30/avgnd))
+    print("ak.array Average rate = {:.4f} GiB/sec".format(nb/2**30/avgpd))
+
+def check_correctness(dtype, random, seed):
+    N = 10**4
+
+    a = ak.randint(0, 2**32, N, dtype=dtype, seed=seed)
+    npa = a.to_ndarray();
+    aka = ak.array(npa)
+    assert np.allclose(a, aka)
```

Comment:
No change requested, I'm just mildly surprised this works on two arkouda arrays. Very cool.

Reply:
Hmm, I am too actually. I wonder what this is doing under the covers. Is there a built in way to compare 2 arkouda arrays?

Reply:
Unfortunately, no. When I do these kinds of comparisons, I usually start with a numpy array, do the round trip, and then compare two numpy arrays. But it's very possible that `np.allclose(a, b)` is just using operators and methods that pdarrays support.

Reply:
I'll switch to to start with the numpy array.